drivers/scheduler/k8s/k8s.go

```
@@ -2841,6 +2834,109 @@ func (k *K8s) destroyBackupObjects(
 	return nil
 }
 
+// EnableVolumeExpansion creates autopilot rules for given context
+func (k *K8s) EnableVolumeExpansion(ctx *scheduler.Context) error {
+	if ctx == nil {
+		return fmt.Errorf("no context provided")
+	}
+	k8sOps := k8sops.Instance()
+	apRule := ctx.ScheduleOptions.AutopilotRule
+	volumes, _ := k.GetVolumes(ctx)
```

any reason to ignore err here?